ion-schema/src/isl/mod.rs

```diff
@@ -945,4 +1007,50 @@ mod isl_tests {
         write!(&mut buf, "{}", range.into()).unwrap();
         assert_eq!(expected, String::from_utf8(buf).unwrap());
     }
+
+    const SKIP_LIST: [&str; 5] = [
+        "ion-schema-schemas/json/json.isl", // the file contains `nan` which fails on equivalence for two schemas
+        "ion-schema-tests/ion_schema_1_0/nullable.isl", // Needs `nullable` annotation related fixes
+        "ion-schema-tests/ion_schema_1_0/schema/import/import_inline.isl", // related to order of types in the schema file
+        "ion-schema-tests/ion_schema_2_0/imports/tree/inline_import_a.isl", // related to order of types in the schema file
+        "ion-schema-tests/ion_schema_2_0/imports/tree/inline_import_c.isl", // related to order of types in the schema file
```

Comment:
Do we have any GitHub issue to track the `nan` issue or the "order of types in a schema file" issue?

Reply:
Created an issue for this, I will add it in the comments for these `SKIP_LIST` items.